src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -190,12 +204,14 @@ public int instanceMain(final String[] argv) {
 
             // Output a one liner about who/where and what software/os we're running on
             try {
-            System.err.println("[" + new Date() + "] Executing as " +
-                                       System.getProperty("user.name") + "@" + InetAddress.getLocalHost().getHostName() +
-                                       " on " + System.getProperty("os.name") + " " + System.getProperty("os.version") +
-                                       " " + System.getProperty("os.arch") + "; " + System.getProperty("java.vm.name") +
-                                       " " + System.getProperty("java.runtime.version") +
-                                       "; Picard version: " + commandLineParser.getVersion());
+                final String msg = String.format(
```

Comment:
I do find this one easier to read. It's a bit harder to figure out which values plug into which slots in the log, but it's easier to see the general structure of the message when it's all on one line at the top.